lib/screens/alerts/informed_entity.ex

```diff
@@ -29,4 +31,20 @@ defmodule Screens.Alerts.InformedEntity do
   def parent_station?(ie) do
     match?(%{stop: "place-" <> _}, ie)
   end
+
+  @spec present_alert_for_route?(t(), Route.id(), Trip.direction() | nil) :: boolean()
+  def present_alert_for_route?(
```

Comment:
I kind of hate this function name if there are any suggestions 😅 

Wanted to make this updatable with the alert changes coming down the pipeline. We can either update this to also accept a list in that case and process the direction_ids of multiple entities.